R/tm_t_pp_basic_info.R

```diff
@@ -265,6 +277,12 @@ srv_t_basic_info <- function(id,
         teal.code::eval_code(as.expression(unlist(my_calls)))
     })
 
+    decorated_table_q <- srv_decorate_teal_data(
+      id = "decorator",
+      data = all_q,
+      decorators = select_decorators(decorators, "table")
+    )
```

Comment:
To display the table if code is copied and run on a separated R session:

```suggestion
      decorators = select_decorators(decorators, "table"),
      expr = table
    )
```